examples/tuning/cta_svm/main.py

```diff
@@ -54,7 +54,7 @@ def __call__(self, data):
     logger.setLevel(args.log_level)
     logger.info(f"\n{pprint.pformat(vars(args))}")
 
-    pipeline_planer = PipelinePlaner.from_config_file(f"{args.tune_mode}_tuning_config.yaml")
+    pipeline_planer = PipelinePlaner.from_config_file(f"examples/tuning/cta_svm/{args.tune_mode}_tuning_config.yaml")
```

Comment:
By default user should run it under the directory of the main function. There are many issues associated with running example scripts in the root directory, primarily due to conflicting data. So, please do not make it a default to run from the root directory.

Alternatively, if you want the flexibility to from from any directory, resolve the `__file__` directory first.


```python
from pathlib import Path
MAINDIR = Path(__file__).resolve().parent
PipelinePlaner.from_config_file(f"{MAINDIR}/{args.tune_mode}_tuning_config.yaml")
```